src/Hourai/Standard/Standard.cs

```diff
@@ -1,300 +1,329 @@
-
-      [Command("sha384")]
-      [Alias("sha-384")]
-      public async Task Sha384(string hash, [Remainder] string input = "") {
-        using (var sha = SHA384.Create()) {
-          await CheckMessage(input, hash, sha);
-        }
-      }
-
-      [Command("sha512")]
-      [Alias("sha-512")]
-      public async Task Sha512(string hash, [Remainder] string input = "") {
-        using (var sha = SHA512.Create()) {
-          await CheckMessage(input, hash, sha);
-        }
-      }
-
-      async Task CheckMessage(string input, string hash, HashAlgorithm hashing) {
-        var url = GetURL(Context, input);
-        var hashCheck = await GetHash(input, url, hashing);
-        if (hash == hashCheck)
-          await Success();
-        else {
-          await RespondAsync($":x: Hash `{hashCheck}` does not match `{hash}`");
-        }
-      }
-    }
-
-  }
-
-}
-
-}
+using Discord;
+using Discord.Commands;
+using Discord.WebSocket;
+using Hourai.Model;
+using Hourai.Preconditions;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Linq;
+using System.Reflection;
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Hourai.Standard {
+
+public partial class Standard : HouraiModule {
+
+  public LogSet Logs { get; set; }
+
+  const ImageFormat ImgFormat = ImageFormat.Auto;
+  const ushort AvatarSize = 1024;
+
+  [Command("echo")]
+  [ChannelRateLimit(3, 1)]
+  [Remarks("Has the bot repeat what you say")]
+  public Task Echo([Remainder] string remainder) => ReplyAsync(Context.Process(remainder));
+
+  [Command("choose")]
+  [ChannelRateLimit(3, 1)]
+  [Remarks("Chooses between several provided choices. Seperated by spaces. Quote choices with spaces in them.")]
+  public Task Choose(params string[] choices) {
+    if (choices.Length <= 0)
+      return RespondAsync($"There is nothing to choose from!");
+    return RespondAsync($"I choose {choices.SelectRandom()}!");
+  }
+  
+  [Command("time")]
+  [ChannelRateLimit(1, 1)]
+  [Remarks("Command relating to calculations of date and time. All times listed compared to Greenwich Mean Time. Daylight savings time NOT shown.")]
+  public Task TimeFrame([Remainder] string country) {
+    return RespondAsync($"WARNING: The time zone readings you may see below do not take into consideration if DST is in effect. If in doubt, double check.")
+   if (country == "USA" || country == "US" || country == "United States") {
+      return RespondAsync($"**-05:00:** Eastern Time Zone - New York, Charlotte, Miami" + \r\n + "**-06:00:** Central Time Zone - Omaha, Kansas City, Chicago" + \r\n + "**-07:00:** Mountain Time Zone - Denver, Helena" + \r\n + "**-08:00:** Pacific Time Zone - San Francisco, Seattle, Las Vegas" + \r\n + "**-09:00:** Alaska Time Zone - Anchorage" + \r\n + "**-10:00:** Hawaii Time Zone - Honolulu, Aleutian Islands")

```

Comment:
Instead of repeated concatenation, use StringBuilder instead. (System.Text.StringBuilder).

Use a variable to hold the value instead, and do one final RespondAsync at the end instead in each clause.